src/codec/RowWriterV2.cpp

```diff
@@ -911,12 +912,19 @@ std::string RowWriterV2::processOutOfSpace() noexcept {
                reinterpret_cast<void*>(&strLen),
                sizeof(int32_t));
     }
-
+    // reset timestamp.
+    auto ts = time::WallClock::fastNowInMicroSec();
+    temp.append(reinterpret_cast<char*>(&ts), sizeof(int64_t));
```

Comment:
We could set ts only once before line 940? Otherwise, we will set once here once in line 926.

Reply:
Good suggestion. and fixed the bug for `checkUnsetFields`